### PR TITLE
[Keyboard] Added Handwired 87 keys

### PR DESCRIPTION
**Pull request by KD-MM2**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Please don't close and re-open PRs: #17622

Also, 
> Yeah, I'm really not comfortable about the name here. ProMicro is a name that technically belongs to SparkFun. And as this isn't a board that is released by them, having it as part of the name is at best sketchy.

File: keyboards/handwired/hwpm87/readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Handwired 87 keys
+
+![hwpm87](https://i.imgur.com/DLqdtAW.jpeg)
```
Comment (drashna) [inline keyboards/handwired/hwpm87/readme.md:3]:
```suggestion
![hwpm87](https://i.imgur.com/DLqdtAWh.jpeg)
```

File: keyboards/handwired/hwpm87/info.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "KD-MM2",
+    "keyboard_name": "promicro87",
```
Comment (drashna) [inline keyboards/handwired/hwpm87/info.json:3]: Should be changed

File: keyboards/handwired/hwpm87/readme.md (hunk 1 above)
Comment (KD-MM2) [reply keyboards/handwired/hwpm87/readme.md:3]: > 

Suggested change is a wrong image link.

File: keyboards/handwired/hwpm87/readme.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Second, if you are using a Capslock LED, define LED pin in: [config.h#34](https://github.com/KD-MM2/qmk_firmware/blob/hwpm87/keyboards/hwpm87/config.h#L34),
+and check the LED_PIN_ON_STATE if you get reversed led state: [config.h#35](https://github.com/KD-MM2/qmk_firmware/blob/hwpm87/keyboards/hwpm87/config.h#L35)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hwpm87:default
+
+Flashing example for this keyboard:
+
+    make hwpm87:default:flash
```
Comment (zvecr) [inline keyboards/handwired/hwpm87/readme.md:25]:
```suggestion
    make handwired/hwpm87:default

Flashing example for this keyboard:

    make handwired/hwpm87:default:flash
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/handwired/hwpm87/rules.mk [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/handwired/hwpm87/rules.mk:18]:
```suggestion
```

File: keyboards/handwired/hwpm87/config.h [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    KD-MM2
+#define PRODUCT         HWPM87
```
Comment (drashna) [inline keyboards/handwired/hwpm87/config.h:13]:
```suggestion
```

File: keyboards/handwired/hwpm87/info.json [hunk 6]
```diff
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "KD-MM2",
+    "keyboard_name": "hwpm87",
+    "maintainer": "KD-MM2",
+    "url": "",
```
Comment (drashna) [inline keyboards/handwired/hwpm87/info.json:5]:
```suggestion
    "url": "",
    "usb": {
        "vid": "0xFEED",
        "pid": "0x0001",
        "device_version": "0.0.1"
    },
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**